Base 1.4. Верстка страницы регистрации

tUnknownLegend opened the pull request.

Liza1040 committed:
Добавила страницу регистрации

tUnknownLegend committed:
исправления для .form__input

+border-left-with. Так же цвета вынесены в константы

tUnknownLegend committed:
сделана основа для хедера

Нужно привести css в адекватный вид, расположить кнопки равнее, перекрасить svg и добавить лого в hi-res

Liza1040 committed:
Поправила header: выровнила элементы, поправила стили

thelvv committed:
Добавлена страница логина и частично главная, шаблоны распределены по файлам

tUnknownLegend committed:
главная страница готова

так же добавлен цвет фона в логине и регистрации в боковых дивах

tUnknownLegend committed:
добавлены теги header и footer, длинна страницы теперь своя у каждой страницы

tUnknownLegend committed:
исправление ширины для поля поиска

tUnknownLegend committed:
страница регистрации

Liza1040 committed:
Поправила стили с учетом базовых стилей кнопки, цветов и шрифтов

tUnknownLegend committed:
унификация css классов

merge from Base-1.2_html_make_up

DPeshkoff commented on signup.html:
```diff
@@ -0,0 +1,116 @@
+                            <div class="form__name text-normal-default">
+                                Почта
+                            </div>
+                            <input class="form__input" type="email" name="email" placeholder="mail@website.com"
```
А почему такой плейсхолдер?

DPeshkoff commented on signup.html:
```diff
@@ -0,0 +1,116 @@
+                            <input class="form__input" type="password" name="repeat_password"
+                                   placeholder="Повторите пароль" maxlength="16">
+                        </div>
+                        <input type="submit" class="button-default form__submit" value="Зарегистрироваться">
```
Здесь, кстати, к кнопке есть стиль на hover, а в header нету. Обсуждали это с ментором по интерфейсам?

DPeshkoff commented on signup.html:
```diff
@@ -0,0 +1,116 @@
+    <div class="paint-background"></div>
+</div>
+</body>
+</html>
```
\n

DPeshkoff commented on signup.css:
```diff
@@ -0,0 +1,41 @@
+}
+
+.content {
+    height: 91vh;
```
почему 91 и почему vh

DPeshkoff commented on signup.css:
```diff
@@ -0,0 +1,41 @@
+    align-items: center;
+}
+
+.right {
+}
```
🤔

DPeshkoff reviewed (changes requested):
Основное ревью [здесь](https://github.com/frontend-park-mail-ru/2022_2_Base/pull/2), сюда отписал конкретно по странице

Непонятно, что в правом блоке - но там и в макете не определились)

DPeshkoff commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Base-1.4: Сверстать signup страницу](https://trello.com/c/e9RqH7Gj/9-base-14-%D1%81%D0%B2%D0%B5%D1%80%D1%81%D1%82%D0%B0%D1%82%D1%8C-signup-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D1%83)

tUnknownLegend committed:
убраны пустые дивы, исправлена строка поиска

по возможности выбраны кратные размеры, выравнены и уравнены в размере правые кнопки в хедере

tUnknownLegend committed:
merge from Base-1.2_html_make_up

tUnknownLegend committed:
Добавлен скрин верстки
